src/scripts/site.js

```diff
@@ -706,35 +680,87 @@ var AppInfo = {};
+                    "sortable",
+                    "libjass"
+                ]
             },
             urlArgs: urlArgs,
             paths: paths,
             onError: onRequireJsError
         });
-        requirejs.onError = onRequireJsError;
 
         // Expose jQuery globally
         require(["jQuery"], function(jQuery) {
             window.$ = jQuery;
             window.jQuery = jQuery;
         });
 
+        require(["css!css/site"]);
+
+        // define styles
+        // TODO determine which of these files can be moved to the components themselves
+        define("material-icons", ["css!css/material-icons/style"], returnFirstDependency);
+        define("systemFontsCss", ["css!css/fonts"], returnFirstDependency);
+        define("systemFontsSizedCss", ["css!css/fonts.sized"], returnFirstDependency);
+        define("scrollStyles", ["css!css/scrollstyles"], returnFirstDependency);
+        define("dashboardcss", ["css!css/dashboard"], returnFirstDependency);
+        define("programStyles", ["css!" + componentsPath + "/guide/programs"], returnFirstDependency);
+        define("listViewStyle", ["css!" + componentsPath + "/listview/listview"], returnFirstDependency);
+        define("formDialogStyle", ["css!" + componentsPath + "/formdialog"], returnFirstDependency);
+        define("clearButtonStyle", ["css!" + componentsPath + "/clearbutton"], returnFirstDependency);
+        define("cardStyle", ["css!" + componentsPath + "/cardbuilder/card"], returnFirstDependency);
+        define("flexStyles", ["css!" + componentsPath + "/flexstyles"], returnFirstDependency);
+
+        // define legacy features
+        // TODO delete the rest of these
+        define("fnchecked", ["legacy/fnchecked"], returnFirstDependency);
+        define("legacyDashboard", ["legacy/dashboard"], returnFirstDependency);
+        define("legacySelectMenu", ["legacy/selectmenu"], returnFirstDependency);
+
+        // there are several objects that need to be instantiated
+        // TODO find a better way to do this
+        define("appFooter", [componentsPath + "/appfooter/appfooter"], returnFirstDependency);
+        define("appFooter-shared", ["appFooter"], createSharedAppFooter);
+
+        // TODO pull apiclient out of this repository
+        define('events', [bowerPath + "/apiclient/events"], returnFirstDependency);
+        define('credentialprovider', [bowerPath + "/apiclient/credentialprovider"], returnFirstDependency);
+        define('connectionManagerFactory', [bowerPath + "/apiclient/connectionmanager"], returnFirstDependency);
+        define('appStorage', [bowerPath + "/apiclient/appStorage"], returnFirstDependency);
+        define("serversync", [bowerPath + "/apiclient/sync/serversync"], returnFirstDependency);
+        define("multiserversync", [bowerPath + "/apiclient/sync/multiserversync"], returnFirstDependency);
+        define("mediasync", [bowerPath + "/apiclient/sync/mediasync"], returnFirstDependency);
```

Comment:
are these three `-sync` things needed?

Reply:
I tried removing some of these but was getting a ton of errors, I'll look into it more once we bundle the API client during build.